docs/cluster_up_down.md

```diff
@@ -367,30 +373,6 @@ To return to the regular `developer` user, login as that user:
 oc login -u developer
 ```
 
-## Docker Machine
-
-By default, when `--create-machine` is used to create a new Docker machine, the `oc cluster up` command will use the
```

Comment:
Per @csrwng we are deprecating `--create-machine`. Instructions for setting the docker machine driver are covered by the external Docker machine links (see above).